backend/src/users/users.service.ts

```diff
@@ -51,24 +35,23 @@ export class UsersService {
   async getAllUsers() {
     const users = await this.prisma.user.findMany({
       select: {
-        // TODO: id is necessary?
-        id: true,
         nickname: true,
         ladderPoints: true,
-        avatar: true,
+        avatarPath: true,
         createdAt: true
       }
     });
     return users;
   }
 
   //TODO: Use guard to verify authorization
-  async deleteUser(paramName: string, nickname: string) {
+  async deleteUser(paramName: string, nickname: string, avatarPath: string) {
```

Comment:
Can't we call deleteUser only with userId or nickname?
Seems tedious to retrieve every resources to be deleted before calling this function.